README.md

```diff
@@ -1,6 +1,62 @@
 # ONNX.js Demo
 
-Coming soon...
+ONNX.js demo is an interactive demo portal showing real use cases running [ONNX.js](https://github.com/Microsoft/onnxjs) in VueJS. It currently supports four examples for you to quickly experience the power of ONNX.js. 
+
+The demo is available here. 
+
+## Use Cases
+
+The demo provides four scenarios based on four different ONNX pre-trained deep learning models. 
+
+### 1. ResNet-50
+
+[ResNet-50](https://github.com/onnx/models/tree/master/models/image_classification/resnet) is a highly-accurate deep convolutional network for image classification. It is trained on 1000 pre-defined classes. In the demo, you can select or upload an image and see which category it's from.
+
+### 2. Squeezenet
```

Comment:
I think Squeezenet might have to come before Resnet now that the order has changed in the demo...